docs/src/GettingStarted/Terminology.md

```diff
@@ -0,0 +1,85 @@
+# Terminology
+
+The `ClimateMachine` documentation uses terminology from several disciplines. Below are some definitions of some of these terms. Please let us know via a comment if there are others that you find confusing, and we will add them.
```

Comment:
I have an impression that we are not adding a comment session for this release. Shall we change it to "via email/issue"?

Reply:
good point! Ill fix this.